lab_3_generate_by_ngrams/main.py

```diff
@@ -159,6 +212,19 @@ def _postprocess_decoded_text(self, decoded_corpus: tuple[str, ...]) -> Optional
 
         In case of corrupt input arguments, None is returned
         """
+        if not isinstance(decoded_corpus, tuple) or decoded_corpus == ():
+            return None
+        str_text = ''
+        for element in decoded_corpus:
+            if element.isalpha():
+                if str_text == '':
```

Comment:
Empty sequences are mapped to False anyway

Reply:
Done